.eslintrc.json

```diff
@@ -77,11 +76,10 @@
         "null": "ignore"
       }
     ],
+    "@typescript-eslint/no-unused-expressions": "off",
+    "@typescript-eslint/no-require-imports": "off",
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
```

Comment:
We want to add in the `eslint/no-duplicate-imports`, right? (linked [here](https://eslint.org/docs/latest/rules/no-duplicate-imports))